epub33/overview/index.html

```diff
@@ -551,39 +553,29 @@ <h2>Semantic Markup</h2>
 					content.</p>
 
-				<p>EPUB 3 includes the <a data-cite="epub-33#sec-epub-type-attribute"><code>epub:type</code></a>
-					attribute, which is meant to be functionally equivalent to the W3C Role Attribute
-					[[Role-Attribute]]. This attribute allows any element in an XHTML Content Document to include
-					additional information about its purpose and meaning within the work, using controlled vocabularies
-					and terms. Refer to the section on <a data-cite="epub-33#app-structural-semantics">Expressing
-						Structural Semantics</a> [[EPUB-33]] for more information.</p>
-
-				<p class="ednote"> With the latest resolution of the WG making the structural vocabulary a note, one may
-					wonder whether referring to epub:type should stay in the Overview document... </p>
-
 			</section>
 
 			<section id="sec-access-layout">
 				<h2>Dynamic Layouts</h2>
 
-				<p>The design center of EPUB is dynamic layout: content is typically intended to be formatted on the fly
-					rather than being typeset in a paginated manner in advance (i.e., expecting a particular sized
-					"page"). This core capability is useful, for example, for optimizing rendering onto different sized
-					device screens or window sizes, and it facilitates and simplifies content accessibility.</p>
+				<p>TAt its core, EPUB is designed for dynamic layout: content is typically intended to be formatted on
+					the fly rather than being typeset in a paginated manner in advance. This core capability is useful
+					for optimizing rendering onto different-sized device screens or window sizes, and it facilitates
+					and simplifies content accessibility.</p>
 
-				<p>While it is possible to incorporate more highly formatted content in EPUB — for example via bitmap
+				<p>While it is possible to incorporate more highly-formatted content in EPUB — for example via bitmap
 					images or SVG graphics, or even use of CSS explicit positioning and/or table elements to achieve
 					particular visual layouts — EPUB Creators are strongly discouraged from utilizing such techniques.
-					They are not reliable in EPUB since many Reading Systems render content in a paginated manner rather
-					than creating a single scrolling <a>Viewport</a> and since each Reading System might define its own
-					pagination algorithm. If these techniques are necessary to convey the content of the publication,
-					consider including <a data-cite="epub-33#sec-foreign-restrictions">fallbacks</a> [[EPUB-33]] (e.g.,
+					These techniques are not reliable in EPUB since many Reading Systems render content in a
+					paginated manner rather than creating a single scrolling <a>Viewport</a> and since each Reading System
+					might define its own pagination algorithm. If these techniques are necessary to convey the content of the
+					publication, consider including <a data-cite="epub-33#sec-foreign-restrictions">fallbacks</a> [[EPUB-33]] (e.g.,
```

Comment:
Should we be saying to use fallbacks here when the spec cautions strongly against them?

Reply:
Sigh... Actually, this section completely ignores FXL; this section essentially says "yes, there is FXL but do not use them".

I guess the best is to remove this sentence altogether, and also remove the section on fallbacks below. (For the time being I put it in comments, in case we want to revive it later). If we believe it is important, a much cleaner approach would be to add a new sectio on FXL altogether at some point.